### ISSUE TITLE
Cargo rebuilds project every time since 81688bc

### ISSUE THREAD

**Issue by ama0** [kvark/froggy#26]
Since 81688bc every `cargo [build|run] [--release]` rebuilds the whole project that uses froggy, including froggy itself.
* 81688bc introduced `skeptic` to check code examples in `README.md`

Steps to reproduce:
```shell
$ cargo new --bin froggy-testing
$ cd froggy-testing
```
Edit Cargo.toml:
```toml
[package]
name = "froggy-testing"
version = "0.1.0"
authors = ["anton"]

[dependencies]
froggy = "0.2.0"
```
Edit src/main.rs:
```rust
extern crate froggy;

fn main() {
    println!("Hello, world!");
}
```
Now build and run
```shell
anton@blues:~/Projects/froggy-testing$ cargo run
   Compiling froggy v0.2.0
   Compiling froggy-testing v0.1.0 (file:///home/anton/Projects/froggy-testing)
    Finished dev [unoptimized + debuginfo] target(s) in 0.66 secs
     Running `target/debug/froggy-testing`
Hello, world!
anton@blues:~/Projects/froggy-testing$ cargo run
   Compiling froggy v0.2.0
   Compiling froggy-testing v0.1.0 (file:///home/anton/Projects/froggy-testing)
    Finished dev [unoptimized + debuginfo] target(s) in 0.66 secs
     Running `target/debug/froggy-testing`
Hello, world!
```

**Comment by kvark** [comment]
"Fixed" by #31

**Comment by bozaro** [comment]
Root cause: https://github.com/rust-lang/cargo/issues/4213

You can workaround this issue by simply adding empty `README.md.skt.md` file.
